### DISCUSSION TITLE
Feature: Add ability to see group names in the register

### DISCUSSION THREAD

**Discussion by irandamay** [actualbudget/releases#196] [Feature ideas]
It would be nice to see what group something came from in the account registers in addition to the category, for categories with the same name, e.g. I have a category called "Food" under my "Flexible" monthly bills group (for people food) and "Pets" for the dogs, but in the registers I can't easily scan and see which "Food" something is applying to.  

I ended up duplicating the group name in the category name for things like that, but it doesn't look very nice in the actual budget list.

**Comment by Deguello** [comment]
Agreed. I have lists of names of people under Christmas and Reimbursements. I ended up changing Gas to Fuel for my car because I also have Gas under utilities.

**Comment by MikeBishop** [comment]
I have an "Other" under multiple groups, and I have both an income and spending category names "Gifts."

**Comment by jlongster** [comment]
Agreed that we need this. I've moved this to discussions as it's a feature request (which ideally would live here).

I've been hesitant to do this because I really don't like the noise of prefixing all the categories with the group name -- you're paying the cost of the group names for all categories when only 1 or 2 are duplicated. One thing I've been thinking of is reversing it so it's `CategoryName - GroupName` and the `GroupName` part would be colored slightly differently to separate it. There's a chance the group name would be cut off so you wouldn't see it, but you have the same problem the other way around. (For me, some of my group names are long so cutting of the cat name would occur more frequently if the group name was prefixed)

Hopefully you'd see enough of the group name to disambiguate it.
